blivet/devicetree.py

```diff
@@ -144,6 +142,14 @@ def devices(self):
 
         return devices
 
+    @property
+    def names(self):
+        """ List of devices names """
+        lv_info = list(lvs_info.cache.keys())
+        names = [dev.name for dev in self._devices] + self._hidden
```

Comment:
`self._hidden` is a list of devices just like `self._devices`. I think you want `+ self._hidden` to be inside the list comprehension rather than outside it.